### DISCUSSION TITLE
What about biomejs instead of prettier?

### DISCUSSION THREAD

**Discussion by frederikhors** [sveltejs/kit#12855] [Q&A]
I tried to search and I didn't find anything: but has anyone ever thought of switching from prettier to [`biome`](https://biomejs.dev/)?

`biome` finishes in 1 second what prettier finishes in 30 on my project!

**Comment by david-plugge** [comment]
Biome does provide way better DX bit i think svelte and other HTML supersets are not yet fully supported

**Reply by frederikhors** [reply]
Yeah, I tried and opened this: https://github.com/biomejs/biome/issues/4365. But this is the only issue I had. And man, the speed: AWESOME! 1s instead of 30+s.

**Reply by david-plugge** [reply]
You definately see the power of rust here, the future is bright ;)
But what about linting? I dont think it works inside svelte files yet
